modules/serverless-deprecated-removed-features.adoc

```diff
@@ -54,5 +58,33 @@ For the most recent list of major functionality deprecated and removed within {S
 |Removed
 |Removed
 |Removed
+|Removed
```

Comment:
In both TP/GA and Deprecated tables, we add new entries from the top, not from the bottom.

Reply:
Okay, noted!